brave/brave6/src/main/java/com/linecorp/armeria/client/brave/ArmeriaHttpClientParser.java

```diff
@@ -72,7 +72,7 @@ public void parse(brave.http.HttpRequest request, TraceContext context, SpanCust
             return;
         }
 
-        span.tag(SpanTags.TAG_HTTP_HOST, firstNonNull(ctx.authority(), "UNKNOWN"))
+        span.tag(SpanTags.TAG_HTTP_HOST, firstNonNullException(ctx::authority, "UNKNOWN"))
```

Comment:
Q) Did you change this because some were tests broken?

Reply:
Yes, a test case `testEmptyEndpointTags()` in `BraveClientTest.class` failed.
```java
void testEmptyEndpointTags() {
    ...
    // expected: "UNKNOWN"
    // but was: null
    assertThat(span.tag("http.host")).isEqualTo("UNKNOWN"); <--- Assertion Error
}
```